documentation/binding-data/tutorial-flow-components-binder-load.asciidoc

```diff
@@ -4,19 +4,21 @@ order: 3
 layout: page
 ---
 
-ifdef::env-github[:outfilesuffix: .asciidoc]
-== Loading from and Saving to Business Objects
+= Loading From and Saving To Business Objects
 
-Once all bindings have been set up, you are ready to actually fill the bound UI components with data from your business object. Changes can be written to the business object automatically or manually.
+Once your bindings are set up, you are ready to fill the bound UI components with data from your business objects. 
 
-Writing the changes automatically when the user makes any change through the UI is often the most convenient option, but it might have undesirable side effects – if some other part of the application is also using the same instance, then that part might show changes before the user has clicked the save button.
-To prevent that, you either need to use a copy of the edited object or use manual writing to only update the object when the user wants to save.
+Changes can be written to business objects manually or automatically. 
 
-=== Manual Reading and Writing
+[NOTE]
```

Comment:
This note is about "Reading and Writing Automatically". That is the most convenient option and suggested for most use case, but for some reason it is presented after the manual options. While we are on this, let's change the order and then we can just acknowledge this in the beginning of that part.

Reply:
I have changed the order. Let me know if it needs anything more.